### ISSUE TITLE
Broken / wrong links when no paid plan is configured

### ISSUE THREAD

**Issue by dogweather** [TryGhost/Ghost#15360]
### Issue Summary

As a new Ghost user, setting up a blog, I'm not ready to connect to Stripe and configure the paid plans for members. Several UI bugs occur for site visitors in this state:

When a visitor signs up as unpaid member and there are no configured paid plans,

* They still see **Sign up** links (although signed up and logged in) Casper v5.3.1 and Journal v1.0.0 and probably others.
* In some themes, they get **Upgrade** buttons which display a broken dialog (Journal v1.0.0).

## Here, a signed-in member is still shown the link to "Sign up":

<img width="1136" alt="Screen Shot 2022-09-02 at 8 18 00 PM" src="https://user-images.githubusercontent.com/150670/188252673-f5c1bacc-ee60-4210-9861-c1b51ea73596.png">

## Here, a signed-in member is shown the upgrade button leading to a broken dialog:

<img width="331" alt="Screen Shot 2022-08-31 at 3 18 45 AM" src="https://user-images.githubusercontent.com/150670/188252726-34d13651-b49a-4fd4-9f94-dfba51bafd09.png">

<img width="536" alt="Screen Shot 2022-08-31 at 3 30 01 AM" src="https://user-images.githubusercontent.com/150670/188252739-410bb14a-54e0-4613-aa34-f8cd7607f6fc.png">

### Steps to Reproduce

1. Create new site with Journal theme.
2. Sign up as free tier member.
3. Browse to About page, see the Upgrade button.
4. See the Signup links in the footers and elsewhere.

### Ghost Version

Ghost(Pro)

### Node.js Version

unknown

### How did you install Ghost?

Pro

### Database type

Other

### Browser & OS version

Safari / current

### Code of Conduct

- [X] I agree to be friendly and polite to people in this repository

**Comment by cathysarisky** [comment]
It looks like there's an error in index.hbs for this theme - that text comes from lines 88-89, and results if the user is a member but not a paid member. The theme needs to be checking if there are paid tiers available and not rendering code here if there aren't..

**Comment by dogweather** [comment]
> It looks like there's an error in index.hbs for this theme - that text comes from lines 88-89...

Yep, here's what a forum user found: https://forum.ghost.org/t/editon-theme-bug-on-home-page/32176/10
